.github/workflows/build-documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: build-documentation
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - name: Install prerequisites
+      run: |
+          python -m pip install sphinx==3.4.0 sphinx-prompt sphinx_substitution_extensions sphinx_book_theme
+    - name: Builds
+      run: |
+          mkdir html
+          sphinx-build -b html documentation/library_guide/ build/
+    - name: Archive build directory
+      uses: actions/upload-artifact@v1
+      with:
+          name: html-documentation
```

Comment:
You may want to consider using the project name in artifacts name for convenience. Or maybe even a short SHA.
I'd ask @ValentinaKats for clarifications.

To me it will look a bit better in Downloads folder.

Reply:
I think something like onedpl-html-docs-<sha> is a good option and SHA can be short. @dcbenito, what do you think?

Reply:
That sounds fine to me.

Reply:
Updated